project3/styles.css

```diff
@@ -0,0 +1,321 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #163346;
+}
+
+form {
+    background-color: #1e2019;
+    max-width: 70%;
+    margin-right: auto;
+    margin-left: auto;
+    padding: 20px;
+
+    display: block;
+    position: relative;
+    bottom: 50px;
+}
+
+label {
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+    color: #cececd;
+}
+
+
+legend {
+    font-family: "Benchnine", monospace;
+    font-size: 22px;
+    color: #cececd;
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+}
+
+section {
+    background-color: #1e2019;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 70%;
+}
+
+.input,
+.textarea,
+.select {
+    font-family: inherit;
+    font-size: inherit;
+}
+
+
+.age {
+    padding-bottom: 30px;
+}
+
+fieldset {
+    border: 0px;
+    padding: 0px;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 70%;
+    display: inline-block;
+}
+
+select {
+    cursor: pointer;
+    width: 100%;
+    height: 70px;
+    font-family: "Benchnine", monospace;
+    font-size: 18px;
+    text-align: center;
+    color: #1e2019;
+
+}
+
+span {
+    display: block;
+}
+
+.shield {
+    width: 240px;
+    height: 239px;
+    display: block;
+    position: relative;
+    top: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #cececd;
+}
+
+.icons {
+    padding: 10px;
+    text-align: center;
+}
+
+@media (min-width:900px) {
```

Comment:
Since you're writing mobile-first media queries, you should write your _smallest_ media queries first and grow from there; so you would want 400px rules listed first, then then 900px ones.